Makefile

```
@@ -277,6 +279,9 @@ MANAGED_CLUSTER_NAMESPACE ?= $(MANAGED_CLUSTER_NAME)
 
 .PHONY: e2e-test
 e2e-test: e2e-dependencies
+	if [ "$(UPSTREAM_TEST)" = "true" ]; then\
```

I usually reference this styleguide for bash: https://google.github.io/styleguide/shellguide.html#s6.3-tests, which would suggest
```suggestion
	if [[ "$(UPSTREAM_TEST)" == "true" ]]; then\
```
But I think what you have here will work... And it looks like the single bracket syntax is used somewhere else in this file, so it's up to you if you want to change it in all the places or keep it like this.

---

@JustinKuli The default shell is `sh` for Make, so double brackets won't work since we're not setting the shell otherwise.

---

Oh, wow, good to know